stdlib/std/send.tact

```diff
@@ -18,7 +21,7 @@ struct SendParameters {
     data: Cell? = null;
 }
 
-fun send(params: SendParameters) {
+fun send(params: SendParameters): Int { // returns forward fee
```

Comment:
We should probably introduce some more constants for modes. Here is the `SENDMSG` instruction description:

> Creates an output action and returns a fee for creating a message. Mode has the same effect as in the case of SENDRAWMSG. Additionally +1024 means - do not create an action, only estimate fee. Other modes affect the fee calculation as follows: +64 substitutes the entire balance of the incoming message as an outcoming value (slightly inaccurate, gas expenses that cannot be estimated before the computation is completed are not taken into account), +128 substitutes the value of the entire balance of the contract before the start of the computation phase (slightly inaccurate, since gas expenses that cannot be estimated before the completion of the computation phase are not taken into account).

Reply:
+64 and +128 here are the same as for regular `SENDRAWMSG`